cache/clock_cache.cc

```diff
@@ -43,69 +45,67 @@ ClockHandleTable::~ClockHandleTable() {
 
 ClockHandle* ClockHandleTable::Lookup(const Slice& key, uint32_t hash) {
   uint32_t probe = 0;
-  int slot = FindElement(key, hash, probe);
-  return (slot == -1) ? nullptr : &array_[slot];
+  ClockHandle* e = FindSlot(
+      key,
+      [&](ClockHandle* h) {
+        if (h->TryInternalRef()) {
+          if (h->IsElement() && h->Matches(key, hash)) {
+            return true;
+          }
+          h->ReleaseInternalRef();
+        }
+        return false;
+      },
+      [&](ClockHandle* h) { return h->displacements == 0; },
```

Comment:
Is it possible to come up with a intuitively named function, instead of ```h->displacements == 0```? I don't have any good suggestions though.

Reply:
``NothingAfterThisProbe()``?